ov/sas_interconnect.go

```diff
@@ -0,0 +1,142 @@
+package ov
+
+import (
+	"encoding/json"
+
+	"github.com/HewlettPackard/oneview-golang/rest"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"github.com/docker/machine/libmachine/log"
+)
+
+type SasInterconnectList struct {
+	Type        string             `json:"type,omitempty"`        // "Type": "SasInterconnectListV3"
+	URI         utils.Nstring      `json:"uri,omitempty"`         // "uri": "/rest/sas-interconnects?sort=name:asc"
+	Category    string             `json:"category,omitempty"`    // "category": sas-interconnects
+	Start       int                `json:"start,omitempty"`       // "start": 0,
+	Count       int                `json:"count,omitempty"`       // "count": 1,
+	Total       int                `json:"total,omitempty"`       // "total": 1,
+	PrevPageURI utils.Nstring      `json:"prevPageUri,omitempty"` // "prevPageUri": null,
+	NextPageURI utils.Nstring      `json:"nextPageUri,omitempty"` // "nextPageUri": null,
+	Members     []SasInterconnects `json:"members,omitempty"`     // "members": []
+}
+
+type SasInterconnects struct {
+	Type                      string                   `json:"type,omitempty"`                      // "type": "SasInterconnectListV3"
+	URI                       utils.Nstring            `json:"uri,omitempty"`                       // "uri": "/rest/sas-interconnects/TWT732W0CY"
+	Category                  string                   `json:"category"`                            // "category": "sas-interconnects"
+	ETag                      string                   `json:"eTag,omitempty"`                      // "eTag": "463bd328-ffc8-40ae-9603-6136fa9e6e58",
+	Created                   string                   `json:"created,omitempty"`                   // "created": "2018-08-02T15:49:59.963Z",
+	Modified                  string                   `json:"modified,omitempty"`                  // "modified": "2018-12-03T18:26:43.335Z",
+	RefreshState              string                   `json:"refreshState,omitempty"`              // "refreshState": "NotRefreshing"
+	StateReason               string                   `json:"stateReason,omitempty"`               //	"stateReason": null
+	InterconnectLocation      SASInterconnectLocation  `json:"interconnectLocation"`                // "interconnectLocation": {}
```

Comment:
No need to create a new Struct, InterconnectLocation struct exists in interconnect.go file. You can make use of it.